### ISSUE TITLE
Mod DLL Question

### ISSUE THREAD

**Issue by AmyRose1288** [LethalCompany/LethalAPI.TerminalCommands#32]
Some Mods Already Contains LethalAPI.TerminalCommands.dll which seems like the 0.9 version

if i install the V1.0 release,do i need to remove LethalAPI.TerminalCommands.dll or keep both?

**Comment by ShimmyMySherbet** [comment]
Yes, the full release is incompatible with the pre-v1.0, as there were some major breaking changes made for the first major release, including rebranding of the library and namespaces. 

Mods using the old `LethalAPI.TerminalCommands` should update to the `LethalAPI.Terminal` library, and switch to a thunderstore dependency, instead of including the terminal API dll with their mod.

**Comment by AmyRose1288** [comment]
so i need to update code that using LethalAPI.TerminalCommands to LethalAPI.LibTerminal ?

**Comment by ShimmyMySherbet** [comment]
Yes, they have the same API layout, but under different namespaces, and with a few types and methods renamed.
